contracts/script/DeployNodeRegistry.s.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: UNLICENSED
```

Comment:
these headers seem inconsistent

Reply:
Good catch, I updated every license to MIT.